env_config/env.docker

```diff
@@ -96,3 +96,10 @@ N8N_ENCRYPTION_KEY=secret
 CLAMAV_PORT=3310
 CLAMAV_HOST=clamav
 ENABLE_FILE_VIRUS_SCAN=
+
+# ------------------------------------------------------------------------------
+# GCNotify - Email and SMS api
+# ------------------------------------------------------------------------------
+GCNOTIFY_SECRET_API_KEY=
+GCNOTIFY_ONBOARDING_REQUEST_EMAIL_TEMPLATE=
+GCNOTIFY_ONBOARDING_REQUEST_SMS_TEMPLATE=
```

Comment:
General comment: If these values aren't really secret, then we can list them here (one less thing for a developer to have to look up).

The secret api key is obviously sensitive, but the templates are presumably not?